core/optionsx.h

```diff
@@ -497,7 +497,8 @@ OPTION_COMMAND(bool, opt_memory, false, "opt_memory",
                "enable memory savings at potential loss in performance", STATIC,
                OP_PCACHE_NOP)
 
-PC_OPTION_INTERNAL(bool, bb_prefixes, "give all bbs a prefix")
+PC_OPTION_DEFAULT_INTERNAL(bool, bb_prefixes, IF_AARCH64_ELSE(true, false),
```

Comment:
This option cannot be disabled for A64, so it seems best to add an options.c rule to prevent that.

Reply:
Done.